src/util/index.js

```diff
@@ -1,25 +1,75 @@
 import axios from "axios";
- 
+
+// ``** GET **``
+
+// GET params example
+const params = {
+  params: {
+    ID: 12345,
+  },
+};
+
 // note: not used, but could be used with GET with params
- const getData = async (url, params) => {
-    try {
-      let res = await axios.get(url, params);
-      let data = await res.data;
-      return data;
-    } catch (error) {
-      console.log(error, `error - getData in ${url} route`);
-    }
-  };
-
-  const getAllData = async (url) => {
-    try {
-      let res = await axios.get(url);
-      let data = await res.data;
-      return data;
-    } catch (error) {
-      console.log(error, `error - getAllData in ${url} route`);
-    }
-  };
-
-
-  export {getData, getAllData};
\ No newline at end of file
+const getData = async (url, params) => {
+  try {
+    let res = await axios.get(url, params);
```

Comment:
make sure to pass params inside curly braces, such as:  axios.get(url, { params });
With axios we usually have: axios.get(url, config), where config is the object that can hold params, headers and timeout

Reply:
So I see you can write: axios.get(url, { params: { ID: 12345 }}).

Would it also be ok to write:

const config = {
  params: {
    ID: 12345,
  }
};

axios.get(url, config)

Since the "config" variable is an object containing the "params" property?